### PR TITLE
Switch detector to nagios plugin

### PR DESCRIPTION
**Pull request by arnnow**
Switch the detector to nagios.state instead of nagios_state.state.

This will work with the new monitor nagios activated on v5.7.0 of the agent : https://docs.signalfx.com/en/latest/integrations/agent/monitors/nagios.html

Also, enable runbook_url and tip for this detector.

### CODE REVIEW

File: modules/smart-agent_nagios-status-check/conf/readme.yaml [hunk 1]
```diff
@@ -1,47 +1,28 @@
 documentations:
```
Comment (xp-1000) [inline modules/smart-agent_nagios-status-check/conf/readme.yaml:1]: this doc is auto generated you have to edit its underlying yaml configuration in conf directory and run `make doc`

File: modules/smart-agent_nagios-status-check/variables.tf [hunk 2]
```diff
@@ -50,3 +50,14 @@ variable "status_check_lasting_duration_seconds" {
   default     = "900"
 }
 
+variable "runbook_url" {
```
Comment (xp-1000) [inline modules/smart-agent_nagios-status-check/variables.tf:53]: this is the object of another issue and so another PR which should be spread to the entire repo (not only one module) to keep modules homogeneous, same for `tip`

File: modules/smart-agent_nagios-status-check/conf/readme.yaml [hunk 3]
```diff
@@ -51,5 +32,5 @@ notes: |
 
   While SignalFx does not provide `Unknown` severity this module uses the `Major` severity for unknown alerts.
 
-  The metric is named `nagios_state.state`, you need to add an `extraDimensions` to your monitor in order to be 
+  The metric is named `nagios.state`, you need to add an `extraDimensions` to your monitor in order to be
```
Comment (xp-1000) [inline modules/smart-agent_nagios-status-check/conf/readme.yaml:35]: this is now false while monitor takes `service` as mandatory parameter

File: modules/smart-agent_nagios-status-check/conf/readme.yaml [hunk 4]
```diff
@@ -1,47 +1,28 @@
...
-    telegrafParser:
-      dataFormat: nagios
-  - type: telegraf/exec
-    intervalSeconds: 900  
-    command: /usr/local/bin/scripts/check-megaraidsas
-    extraDimensions:
-      script: check_megaraidsas
-    telegrafParser:
-      dataFormat: nagios
+  - type: nagios
```
Comment (xp-1000) [inline modules/smart-agent_nagios-status-check/conf/readme.yaml:20]: may be can you add more examples, like different use cases

File: modules/smart-agent_nagios-status-check/conf/readme.yaml [hunk 5]
```diff
@@ -1,47 +1,28 @@
...
+  intervalSeconds: 180
+  command: /usr/lib/nagios/plugins/check_ntp_time -H pool.ntp.typhon.net -w 0.5 -c 1
+  service: NTP_TIME
+  extraDimensions:
+    sfx_monitored: false
```
Comment (xp-1000) [inline modules/smart-agent_nagios-status-check/conf/readme.yaml:25]: it breaks the alerting so it must not be in the example (or at worst, as a counter example to explain how to collect metric without alert on this check)

**Review by xp-1000 — CHANGES_REQUESTED** [review]

**Comment by xp-1000** [comment]
we also to inform this module requires agent >= 5.7.1

File: modules/smart-agent_nagios-status-check/conf/readme.yaml (hunk 1 above)
Comment (arnnow) [reply modules/smart-agent_nagios-status-check/conf/readme.yaml:1]: I get an error when trying to generate the readme
```
terraform-signalfx-detectors:master ✓ ➭ make readmes modules/smart-agent_nagios-status-check/
./scripts/module/loop_wrapper.sh ./scripts/module/gen_doc.sh
sed: -e expression n°1, caractère 54: Fin d'intervalle invalide
make: *** [Makefile:34 : readmes] Erreur 1
```

File: modules/smart-agent_nagios-status-check/variables.tf (hunk 2 above)
Comment (arnnow) [reply modules/smart-agent_nagios-status-check/variables.tf]: Removed, i will try to create another PR for this

File: modules/smart-agent_nagios-status-check/conf/readme.yaml (hunk 3 above)
Comment (arnnow) [reply modules/smart-agent_nagios-status-check/conf/readme.yaml]: Removed from the doc

File: modules/smart-agent_nagios-status-check/conf/readme.yaml (hunk 4 above)
Comment (arnnow) [reply modules/smart-agent_nagios-status-check/conf/readme.yaml:20]: I added another exemple to the doc

File: modules/smart-agent_nagios-status-check/conf/readme.yaml (hunk 5 above)
Comment (arnnow) [reply modules/smart-agent_nagios-status-check/conf/readme.yaml]: yes, i switch `sfx_monitored` to true

**Comment by arnnow** [comment]
> we also to inform this module requires agent >= 5.7.1

I did add this information to the readme. 
Tell me if that's ok with you

**Review by xp-1000 — APPROVED** [review]
Good, thanks

**Pull request merged**